helm/ph-ee-test/values.yaml

```diff
@@ -1,35 +1,27 @@
 ph-ee-engine:
-  zeebe:
-    broker:
-      contactpoint: "zeebetest-zeebe-gateway:26500"
-
-  #Old zeebe configuration to be replaced with camunda-platform:zeebe/zeebe gateway:<paramname>
-#  zeebe-cluster-helm:
-#    global:
-#      elasticsearch:
-#        host: "ph-ee-elasticsearch"
-#    image:
-#      repository: camunda/zeebe
-#      tag: 1.1.0
-#
-#    clusterSize: "1"
-#    partitionCount: "1"
-#    replicationFactor: "1"
-#    JavaOpts: "-Xms8g -Xmx8g -XX:+UseParallelGC -XX:MinHeapFreeRatio=5 -XX:MaxHeapFreeRatio=10 -XX:MaxRAMPercentage=25.0 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:+PrintFlagsFinal"
-#
-#    elasticsearch:
-#      enabled: false
-#    kibana:
-#      enabled: false
-#
-#    extraInitContainers: |
-#      - name: init-ph-ee-kafka-exporter
-#        image: busybox:1.28
-#        command: ['/bin/sh', '-c']
-#        args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://paymenthub-ee-dev.s3.us-east-2.amazonaws.com/jars/exporter-1.0.0-SNAPSHOT.jar"; ls -al /exporters/']
-#        volumeMounts:
-#        - name: exporters
-#          mountPath: /exporters/
+
+  camunda-platform:
+    global:
+      zeebe: "zeebetest-zeebe"
+    zeebe:
+      broker:
+        contactpoint: "zeebetest-zeebe-gateway:26500"
```

Comment:
this is already available in https://github.com/openMF/ph-ee-env-template/pull/114/files

Reply:
@fynmanoj broker.contactpoint is having overridden value. If we want to have contactpoint value as zeebe-zeebe-gateway then i'll remove it.

Reply:
you have to use the overridefullname property and use it. the word `zeebetest` is coming from the release name and you do not have any control over it when someone else is using it in a different cluster